cdts/ibacm-cos7-aarch64/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: ibacm-cos7-aarch64
+  version: 22.4
+
+source:
+  - url: http://mirror.centos.org/altarch/7/os/aarch64/Packages/ibacm-22.4-1.el7.aarch64.rpm
+    sha256: a7f447298f3fdaa87e3e9896b2720dc0747a7d3361f22e483daa2b789106d3e3
+    no_hoist: true
+    folder: binary
+  # - url: http://vault.centos.org/7.7.1908/os/Source/SPackages/rdma-core-22.4-1.el7.src.rpm
+  #   no_hoist: true
+  #   folder: source
+
+build:
+  number: {{ cdt_build_number|int + 1000 }}
+  noarch: generic
+  binary_relocation: False
+  detect_binary_files_with_prefix: False
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - libibumad-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - libibverbs-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - rdma-core-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+  host:
+    - libibumad-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - libibverbs-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - rdma-core-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - sysroot_linux-aarch64 2.17.*
+  run:
+    - libibumad-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - libibverbs-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - rdma-core-cos7-aarch64 ==22.4 *_{{ cdt_build_number|int + 1000 }}
+    - sysroot_linux-aarch64 2.17.*
+
+test:
+  commands:
+    - echo "it installs!"
+
+about:
+  home: https://github.com/linux-rdma/rdma-core
+  license: GPLv2 or BSD
+  license_family: GPL2
+  license_file: {{ SRC_DIR }}/binary/LICENSE_NOT_FOUND
```

Comment:
@jakirkham When the gen_cdt_recipes.py script ran the first time, it printed out a warning that it could not find a license for this CDT too. So you'll have to add it as well by listing it in the main yaml file, getting the license files, and copying them over etc.

Reply:
I've tried deleting the CDTs and regenerating them. However this line does not appear to be getting replaced. Am I missing something else?

Reply:
Nope it should just work.

Run the gen script one more time. It replaces licenses always.

Reply:
Sorry I'm not sure what's up. Tried again, but am still having the same issue. Included as commits below removing the CDTs in this PR and the results from regenerating.

Reply:
are you working off of master?

Reply:
i'd be better if you forked this PR into a person-based account so I can make changes.

Reply:
Yep, it's based off `master`.

Added you as a collaborator on this repo.

Reply:
I am getting this

```
(base) clarence:cdt-builds beckermr$ git rm -rf cdts/ibacm*
rm 'cdts/ibacm-cos6-x86_64/COPYING.BSD_FB'
rm 'cdts/ibacm-cos6-x86_64/COPYING.BSD_MIT'
rm 'cdts/ibacm-cos6-x86_64/COPYING.GPL2'
rm 'cdts/ibacm-cos6-x86_64/COPYING.md'
rm 'cdts/ibacm-cos6-x86_64/build.sh'
rm 'cdts/ibacm-cos6-x86_64/meta.yaml'
rm 'cdts/ibacm-cos7-aarch64/COPYING.BSD_FB'
rm 'cdts/ibacm-cos7-aarch64/COPYING.BSD_MIT'
rm 'cdts/ibacm-cos7-aarch64/COPYING.GPL2'
rm 'cdts/ibacm-cos7-aarch64/COPYING.md'
rm 'cdts/ibacm-cos7-aarch64/build.sh'
rm 'cdts/ibacm-cos7-aarch64/meta.yaml'
rm 'cdts/ibacm-cos7-ppc64le/COPYING.BSD_FB'
rm 'cdts/ibacm-cos7-ppc64le/COPYING.BSD_MIT'
rm 'cdts/ibacm-cos7-ppc64le/COPYING.GPL2'
rm 'cdts/ibacm-cos7-ppc64le/COPYING.md'
rm 'cdts/ibacm-cos7-ppc64le/build.sh'
rm 'cdts/ibacm-cos7-ppc64le/meta.yaml'
(base) clarence:cdt-builds beckermr$ git rm -rf legacy_cdts/ibacm*
rm 'legacy_cdts/ibacm-cos7-ppc64le/COPYING.BSD_FB'
rm 'legacy_cdts/ibacm-cos7-ppc64le/COPYING.BSD_MIT'
rm 'legacy_cdts/ibacm-cos7-ppc64le/COPYING.GPL2'
rm 'legacy_cdts/ibacm-cos7-ppc64le/COPYING.md'
rm 'legacy_cdts/ibacm-cos7-ppc64le/build.sh'
rm 'legacy_cdts/ibacm-cos7-ppc64le/meta.yaml'
(base) clarence:cdt-builds beckermr$ python gen_cdt_recipes.py --fast
generating CDT recipes...
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.ppc64le_495155b8db.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.ppc64le_495155b8db.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.ppc64le_23c2992f73.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.aarch64_d82ad44ad0.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.aarch64_0909fc71c0.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.aarch64_d82ad44ad0.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.aarch64_d82ad44ad0.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.aarch64_0909fc71c0.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.aarch64_d82ad44ad0.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.aarch64_d82ad44ad0.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.aarch64_0909fc71c0.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.x86_64_a54d2c5ce3.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.x86_64_a54d2c5ce3.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.x86_64_b2366cabf7.rpm!                                                                    
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.x86_64_a54d2c5ce3.rpm!                                                                   
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.x86_64_b2366cabf7.rpm!                                                                    
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.ppc64le_495155b8db.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM libibverbs-22.4-1.el7.ppc64le_495155b8db.rpm!                                                                  
WARNING: could not find a suitable license file in the RPM librdmacm-22.4-1.el7.ppc64le_23c2992f73.rpm!                                                                   
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 18/18 [00:19<00:00,  1.08s/it]
(base) clarence:cdt-builds beckermr$ 
```

what did you see?

Reply:
you need to add licenses for all of those cdts.

Reply:
Interesting. Those are actually from PR ( https://github.com/conda-forge/cdt-builds/pull/10 ) and PR ( https://github.com/conda-forge/cdt-builds/pull/12 ). Happy to update them here though 🙂

Reply:
Ahh right. So you'll need to remove the skips I pointed you to on github and add licenses. The cos6 packages had licenses and so I didn't need to do anything. For w/e reason I didn't make the cos7 ones. Probably because they were not on the channels already, but I don't actually recall.

Reply:
Yeah copy and pasted the licenses.

Though when I run the script again it seems to lose the license files again. So didn't do that. Are there additional changes in the script that you used locally that haven't been upstreamed? Should add it's equally possible I'm doing something wrong 😅

Reply:
I am running it from your branch on your fork. It appears to pause at the end but is doing work. Make sure to let it finish. Also, you need to run `git add *` on the changes before committing. Did you do those things?

Reply:
I've added some more output in the generator script and some extra instructions in https://github.com/conda-forge/cdt-builds/pull/27. LMK if that helps.